### DISCUSSION TITLE
Déployer container wiki avec keycloack pré-configuré

### DISCUSSION THREAD

**Discussion by chrispelmcn** [requarks/wiki#7093] [Help / Questions]
Bonjour, J'essaie de déployer Wiki.js avec keycloack pré-configuré mais dans la documentation je ne vois rien qui explique comment le faire. 
Avez vous deja essayer cela? 
Avec vous une idée comment le faire?
Merci.

**Comment by NGPixel** [comment]
This is not possible.

You can however setup Wiki.js, configure Keycloak and make a backup of the database. You can then use this database to any number of deployments.

**Reply by chrispelmcn** [reply]
Merci
